styles.css

```diff
@@ -1 +1,68 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+.grid-container {
+    background-color: antiquewhite;
+    display: grid;
+    row-gap: 16px;
+    column-gap: 16px;
+
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 50px 240px 100px auto 50px;
+    grid-template-areas:
+    "header aside-first aside-first"
+    "main aside-first aside-first"
+    "main aside-third aside-second"
+    "main aside-third ."
+    "footer footer footer";
+}
+
+@media screen and (max-width: 576px) {
```

Comment:
De toewijzing is 'mobile first' en de twee .grid-containers hadden andersom moeten staan met de @media tag voor een scherm breder dan 576px. Zelf onthoud ik het met de stelregel van 'mobile first'.